CONTRIBUTING.MD

```diff
@@ -0,0 +1,38 @@
+# Contributing Guidelines
+
+We love your input! DDD Perth is dedicated to providing a harassment-free conference experience for everyone, regardless of but not limited to: gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. We do not tolerate harassment of others in any form. By participating in this project, you agree to the DDD Perth [code of conduct](https://dddperth.com/code-of-conduct).
+
+## We Develop with GitHub
+
+We use GitHub to host code, to track issues and feature requests, as well as accept pull requests.
+
+1.  Fork the repo and create your branch from `master`.
+2.  If you've added code that should be tested, add tests.
+3.  Ensure the test suite passes.
+4.  Format your code by running `yarn run format-code`
+5.  Make sure your code lints by running `yarn run lint`.
+6.  Go on and issue that pull request!
```

Comment:
Unsure if we want to mention a build will automatically be run against each PR?

Reply:
Heh, just realised PR builds from forks aren't turned on; it was there previously.

https://docs.microsoft.com/en-us/vsts/build-release/actions/ci-build-github?view=vsts says it's beeen removed for "security hardening" and due back "apil 2018".